packages/web/src/components/apps/workspace/editor/editor.tsx

```diff
@@ -60,17 +54,42 @@ function CodeEditor({
   onChange: (file: FileType, attrs: Partial<FileType>) => void;
 }) {
   const { codeTheme } = useTheme();
+  const editorRef = useRef<HTMLDivElement>(null);
 
   const languageExtension = getCodeMirrorLanguageExtension(file);
   const extensions = languageExtension ? [languageExtension] : [];
 
+  useEffect(() => {
```

Comment:
I'm not sure we want to use refs and an observable pattern here. Concerned this will introduce some jitter.
Can we remove this stuff from the PR? We need to take a pass at looking at options for resizing the code editor but I am suspecting this is not trivial.